installer/scripts/bootstrap.bash

```diff
@@ -1,30 +1,43 @@
 #!/bin/bash
 
+# Set ROS version
+case $1 in
+    kinetic)
+        TUE_ROS_DISTRO=kinetic
+        ;;
+    melodic)
+        TUE_ROS_DISTRO=melodic
+        ;;
+    *)
+        echo "[bootstrap] Unknown ROS distribution $1"
+        exit 1
+        ;;
+esac
+
 # make sure git is installed
 hash git 2> /dev/null || sudo apt-get install --assume-yes git
 
 # Move old environments and installer
-if [[ -e ~/.tue && -z "$CI" ]]
+if [[ -d ~/.tue && -z "$CI" ]]
 then
     FILES=$(find ~/.tue/user/envs -maxdepth 1 -type f)
+    date_now=$(date +%F_%R)
     for env in $FILES
     do
-        mv -f $(cat $env) $(cat $env).$(date +%F_%R)
+        mv -f $(cat $env) $(cat $env).$date_now
     done
-    mv -f ~/.tue ~/.tue.$(date +%F_%R)
+    mv -f ~/.tue ~/.tue.$date_now
 fi
 
 if [[ -z "$CI" ]]
 then
     # Update installer
-    git clone https://github.com/tue-robotics/tue-env.git ~/.tue
+    git clone -b ar13pit-patch/fix-names-scripts-targets https://github.com/tue-robotics/tue-env.git ~/.tue
 fi
 
 # Source the installer commands
 source ~/.tue/setup.bash
-
-# Set ROS version
-TUE_ROS_DISTRO=kinetic
+echo "[bootstrap] Loading tue-env"
```

Comment:
This hardly takes any time, so unnecessary in my opinion. And if you put it there, put it before sourcing it.